libs/ledger/src/protocols/main_chain_rpc_service.cpp

```diff
@@ -496,15 +491,13 @@ MainChainRpcService::State MainChainRpcService::OnSynchronised(State current, St
   FETCH_UNUSED(current);
 
   MainChain::BlockPtr head_of_chain = chain_.GetHeaviestBlock();
-  uint64_t const      seconds_since_last_block =
-      GetTime(fetch::moment::GetClock("default", fetch::moment::ClockType::SYSTEM)) -
-      head_of_chain->timestamp;
 
   // Assume if the chain is quite old that there are peers who have a heavier chain we haven't
   // heard about
-  if (seconds_since_last_block > 100 && head_of_chain->block_number != 0)
+  if (loose_blocks_seen_ > 5)
```

Comment:
Might want some rethinking but it is worth avoiding the case where a single loose block bumps you from this state